docs/source/guides/deployment.rst

```diff
@@ -125,22 +125,35 @@ If they are met, you can move forward with the following command:
 .. code-block:: shell
 
     docker run --name iroha \
-    # External port
     -p 50051:50051 \
-    # Folder with configuration files
     -v ~/Developer/iroha/example:/opt/iroha_data \
-    # Blockstore volume
-    -v blockstore:/tmp/block_store \
-    # Postgres settings
+    -v blockstore:/tmp/block_store \ 
     -e POSTGRES_HOST='some-postgres' \
     -e POSTGRES_PORT='5432' \
     -e POSTGRES_PASSWORD='mysecretpassword' \
     -e POSTGRES_USER='postgres' \
-    # Node keypair name
     -e KEY='node0' \
-    # Docker network name
     --network=iroha-network \
-    hyperledger/iroha:latest
+    hyperledger/iroha:latest.
+
+A little more about what these commands mean:
```

Comment:
```suggestion
A little more about what these options mean:
```

Reply:
At least let us call them lines? I mean, options do not sound right.

Reply:
You can call them lines as well. But these are the *options* to `docker run` command.

Reply:
please check out the latest commit